test/smoketest/smoketest-update.bash

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+set -eux -o pipefail
+
+# Put these somewhere else before running via travis
+export TRAVIS_BUILD_DIR=`git rev-parse --show-toplevel`
```

Comment:
I have not tried running this from Travis yet, but I intend to after figuring out what the python install pieces will look like etc.
For now, I'm just using the same environmental variables, and this was a quick hack to make them easy to set.